### PR TITLE
removed code saturn from workflows, fix #464

### LINKED ISSUE #464: Remove CodeStaturn worlkflow from release
related to the removing the openstack option

### PR DESCRIPTION
**Pull request by TristanWright**

### CODE REVIEW

**Comment by TristanWright** [comment]
(this commit was plucked from the nwchem branch)

File: src/workflows/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 import PyFr         from './pyfr/pyfr-simput';
 import PyFrExec     from './pyfr/pyfr-exec';
 import Visualizer   from './visualizer';
 
 const Workflows = {
-  CodeSaturn,
+  // CodeSaturn,
   PyFr,
   PyFrExec,
   Visualizer,
```
Comment (cjh1) [inline src/workflows/index.js]: I would delete these lines rather than comment them out, I don't think we will be support CodeSaturn in the future.

Comment (TristanWright) [inline src/workflows/index.js]: If that's the case should the codesaturn folder be entirely removed too?

Comment (TristanWright) [inline src/workflows/index.js]: It just contains an introduction, nothing we can re-add in a few minutes, removing it.

Comment (cjh1) [inline src/workflows/index.js]: Yes, the beauty of using a VCS is nothing is really gone, we can always revert this commit in the future.

**Pull request merged**